frontend/src/pages/booking/components/CompleteModal/CompleteModal.tsx

```diff
@@ -2,64 +2,85 @@ import { css } from '@emotion/react';
 import styled from '@emotion/styled';
 import { useNavigate } from 'react-router-dom';
 
-import bookedIcon from '../../../../common/assets/images/bookedIcon.svg';
+import humanIcon from '../../../../common/assets/images/human.svg';
 import Button from '../../../../common/components/Button/Button';
+import MentoringStepper from '../../../../common/components/mentoringStepper/MentoringStepper/MentoringStepper';
 import Modal from '../../../../common/components/Modal/Modal';
 import { PAGE_URL } from '../../../../common/constants/url';
+import { THEME } from '../../../../common/styles/theme';
+import { StatusTypeEnum } from '../../../../common/types/statusType';
 
-import type { BookingResponse } from '../../types/BookingResponse';
+import type { MentoringDetail } from '../../apis/getMentoringDetail';
```

Comment:
p3: type에 대한 경로가 잘못 된 거 같아! 👍

```suggestion
import type { MentoringDetail } from '../../../../common/types/MentoringDetail';
```

Reply:
헉 맞네 지우면서 잘못들어갔나봐 고마워!!

Reply:
수정커밋: [fix: 잘못된 타입 경로 수정](https://github.com/woowacourse-teams/2025-Fit-toring/pull/611/commits/b660e7186026228b4e9fc006f5b36015c0797a02)